### PR TITLE
[service.subtitles.torec] 1.0.8

### PR DESCRIPTION
**Pull request by slmosl**
Hey,

This pull request mainly handles breaking changes in the subtitle provider site

### CODE REVIEW

File: service.subtitles.torec/resources/lib/TorecSubtitlesDownloader.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def _requestSubtitle(self, subID, subURL):
     def getDownloadLink(self, subID, optionID, subURL, persist=True):        
         requestID = self._requestSubtitle(subID, subURL)
         
-        params = {"sub_id" : subID, "code": optionID, "sh" : "yes", "guest" : requestID, "timewaited" : "12"}
+        params = {"sub_id" : subID, "code": optionID, "sh" : "yes", "guest" : requestID, "timewated" : "15"}
```
Comment (phil65) [inline service.subtitles.torec/resources/lib/TorecSubtitlesDownloader.py]: "timewated" typo or correct?

Comment (slmosl) [inline service.subtitles.torec/resources/lib/TorecSubtitlesDownloader.py]: correct,
the provider (Torec) probably made a mistake

Comment (phil65) [inline service.subtitles.torec/resources/lib/TorecSubtitlesDownloader.py]: ok.

**Comment by phil65** [comment]
looks fine to me.

**Pull request merged**